packages/mask/src/plugins/EVM/hooks/useHistory.ts

```diff
@@ -0,0 +1,9 @@
+import { useAsyncRetry } from 'react-use'
+import { EVM_RPC } from '../messages'
+
+export function useHistory(address?: string, tokenId?: string) {
+    return useAsyncRetry(async () => {
+        if (!address || !tokenId) return
```

Comment:
If the history is `type[]` then it should return an array too.

```suggestion
        if (!address || !tokenId) return []
```